packages/orbit-components/src/Tooltip/README.md

```diff
@@ -18,19 +18,20 @@ After adding import into your project you can use it simply like:
-| stopPropagation      | `boolean`       |         | If `true` the click event on children won't bubble. Useful when you use Tooltip inside another clickable element. |
-| renderInPortal       | `boolean`       | `true`  | Optional prop, set it to `false` if you're rendering Tooltip inside a custom portal, defaults to `true`           |
+| Name                 | Type            | Default | Description                                                                                                                                    |
+| :------------------- | :-------------- | :------ | :--------------------------------------------------------------------------------------------------------------------------------------------- |
+| **children**         | `React.Node`    |         | The reference element where the Tooltip will appear.                                                                                           |
+| **content**          | `React.Node`    |         | The content to display in the Tooltip.                                                                                                         |
+| dataTest             | `string`        |         | Optional prop for testing purposes.                                                                                                            |
+| enabled              | `boolean`       | `true`  | Enable render of tooltip                                                                                                                       |
+| block                | `boolean`       | `false` | Whether the children wrapper is inline or block. Useful when children need to take up the entire container width.                              |
+| preferredAlign       | [`enum`](#enum) |         | The preferred align to choose [See Functional specs](#functional-specs)                                                                        |
+| preferredPosition    | [`enum`](#enum) |         | The preferred position to choose [See Functional specs](#functional-specs)                                                                     |
+| removeUnderlinedText | `boolean`       |         | Removes underline on child component, when underline is not desired.                                                                           |
+| size                 | [`enum`](#enum) |         | The maximum possible size of the Tooltip.                                                                                                      |
+| stopPropagation      | `boolean`       |         | If `true` the click event on children won't bubble. Useful when you use Tooltip inside another clickable element.                              |
+| lockScrolling        | `boolean`       | `true`  | Whether to prevent scrolling of the rest of the page while Tooltip is open on mobile. This is on by default to provide better user experience. |
```

Comment:
```suggestion
| lockScrolling        | `boolean`       | `true`  | Whether to prevent scrolling of the rest of the page while Tooltip is open on mobile. This is on by default to provide a better user experience. |
```